core/migrations/replace_0001_initial.py

```diff
@@ -0,0 +1,1466 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.4 on 2018-08-15 21:46
+from __future__ import unicode_literals
+
+import core.models.allocation_strategy
+import core.models.quota
+import core.validators
+from django.conf import settings
+import django.contrib.auth.models
+import django.contrib.postgres.fields.jsonb
+import django.core.validators
+from django.db import migrations, models
+import django.db.migrations.operations.special
+import django.db.models.deletion
+import django.utils.timezone
+import uuid
+
+
+# Functions from the following migrations need manual copying.
+# Move them and any dependencies into this file, then update the
+# RunPython operations to refer to the local versions:
+# core.migrations.0088_set_project_leaders_and_authors
+# core.migrations.0089_projects_to_project__alter_created_by
+# core.migrations.0092_set_unknown_size_instance_status_history

```

Comment:
Would you replace this comment with a short summary of why this migration exists.